scripts/e2e.sh

```diff
@@ -29,7 +29,7 @@ return-pod-exit-code() {
 
 wait-for-pod-ready() {
   local name="${1}"
-  local timeout_secs=180
+  local timeout_secs=15
```

Comment:
Is changing `timeout_secs` part of retrying 5 times? This appears to be independent of that change.